### PR TITLE
Signalr hub handling

### PR DESCRIPTION
**Pull request by andrejp6k**

### CODE REVIEW

File: src/redux/store.ts [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import { configureStore } from '@reduxjs/toolkit';
+import websocketReducer from './websocketSlice';
+
+export const store = configureStore({
```
Comment (kimjetam) [inline src/redux/store.ts:4]: any good resource on react redux?

**Review by kimjetam — APPROVED** [review]

File: src/renderer/components/AssignUserToStudio.tsx [hunk 2]
```diff
@@ -29,14 +30,14 @@ function AssignStudioToUser() {
     const queryString = global.location.search;
     const urlParams = new URLSearchParams(queryString);
     const deviceIdFromParams = urlParams.get('deviceId');
-    setDeviceId(deviceIdFromParams);
+    setDeviceId(deviceIdFromParams || '');
```
Comment (podhorets) [inline src/renderer/components/AssignUserToStudio.tsx:33]: we need undefined handler here. '' will bring us troubles in business logic.

File: src/renderer/components/AssignUserToStudio.tsx [hunk 3]
```diff
@@ -29,14 +30,14 @@ function AssignStudioToUser() {
...
   }, []);
 
-  const handleStudioClick = (studio) => {
+  const handleStudioClick = (studio: any) => {
     setSelectedStudio(studio);
   };
 
-  const handleRoleClick = (role) => {
+  const handleRoleClick = (role: any) => {
```
Comment (podhorets) [inline src/renderer/components/AssignUserToStudio.tsx:40]: can be Roles

File: src/renderer/components/AssignUserToStudio.tsx [hunk 4]
```diff
@@ -29,14 +30,14 @@ function AssignStudioToUser() {
     const queryString = global.location.search;
     const urlParams = new URLSearchParams(queryString);
     const deviceIdFromParams = urlParams.get('deviceId');
-    setDeviceId(deviceIdFromParams);
+    setDeviceId(deviceIdFromParams || '');
   }, []);
 
-  const handleStudioClick = (studio) => {
+  const handleStudioClick = (studio: any) => {
```
Comment (podhorets) [inline src/renderer/components/AssignUserToStudio.tsx:36]: can be Studio.
we can define type of state.

File: src/renderer/components/SelectGame.tsx [hunk 5]
```diff
@@ -19,6 +20,8 @@ function SelectGame() {
 
   const handleGameClick = (gameId: string) => {
     console.log(gameId);
+
+    sendMessage('RegisterTeam', 'gopnik team');
```
Comment (podhorets) [inline src/renderer/components/SelectGame.tsx:24]: on that step we will need deviceId

File: src/renderer/components/AssignUserToStudio.tsx (hunk 2 above)
Comment (podhorets) [reply src/renderer/components/AssignUserToStudio.tsx:33]: we can think about it in next call how we wanna proceed here. and also how we wanna keep deviceId globally across whole app.

**Comment by podhorets** [comment]
Good job Andrej.

**Review by podhorets — APPROVED** [review]

**Pull request merged**